### ISSUE TITLE
Compile failed on step "rake assets:precompile RAILS_ENV=production"

### ISSUE THREAD

**Issue by Jatin-MYST** [chatwoot/chatwoot#4986]
**Describe the bug**

Tried to follow the self hosted linux VM setup steps.
Compile failed on step "rake assets:precompile RAILS_ENV=production". See log file.
[chatwoot-setup.log](https://github.com/chatwoot/chatwoot/files/9060639/chatwoot-setup.log)

I have tried terminating and trying on another of the same instance but no luck - getting the same error.

**To Reproduce**

Brand new t3a.xlarge EC2 instance on AWS.
1. switch to root
2. cd to root home
3. wget https://get.chatwoot.app/linux/install.sh
4. chmod +x install.sh
5. ./install.sh master

**Environment**

Ubuntu 20.04 - 4 core 16gb ram
Brand new t3a.xlarge EC2 instance on AWS.

**Comment by pranavrajs** [comment]
Relevant part in the logs:

```
Done in 13.77s.
/usr/local/rvm/gems/ruby-3.0.4/gems/sassc-2.4.0/lib/sassc/engine.rb:43: [BUG] Segmentation fault at 0x0000000000000000
ruby 3.0.4p208 (2022-04-12 revision 3fa771dded) [x86_64-linux]
```

**Comment by vishnu-narayanan** [comment]
Related #4743

**Comment by vishnu-narayanan** [comment]
@Jatin-MYST This has been occurring randomly on our end as well. Could you try this out on a c-type instance? Say `c5.xlarge`?

**Comment by Jatin-MYST** [comment]
I tried on c-type instance and it worked. Thanks for the suggestion @vishnu-narayanan . 
Don't quite understand why it would work on one type of instance and not the other.

Switched it back to t3a after it was done setting up though. Will that be a problem?

**Comment by vishnu-narayanan** [comment]
@Jatin-MYST T-type instances are burstable instances. The CPU credits might have been used up during the compilation stage. That might be the reason.

**Comment by Jatin-MYST** [comment]
@vishnu-narayanan There is no limit on CPU credits.
Oh well its installed now for me, so upto you guys if you want the issue to stay open for a permanent fix.

**Comment by vishnu-narayanan** [comment]
@Jatin-MYST Yeah, you are right. T3 instances now have unlimited mode. I will try to reproduce this issue.
